imagehash/__init__.py

```diff
@@ -116,10 +116,10 @@ def average_hash(image, hash_size=8):
 def phash(image, hash_size=32):
 	image = image.convert("L").resize((hash_size, hash_size), Image.ANTIALIAS)
 	pixels = numpy.array(image.getdata(), dtype=numpy.float).reshape((hash_size, hash_size))
-	dct = scipy.fftpack.dct(pixels)
-	dctlowfreq = dct[:8, 1:9]
-	avg = dctlowfreq.mean()
-	diff = dctlowfreq > avg
+	dct = scipy.fftpack.dct(scipy.fftpack.dct(pixels, axis=0), axis=1)
+	dctlowfreq = dct[:8, :8]
```

Comment:
Thanks guys for your replies. I just want to use it, that is all. Decisions are yours and I'll trust you on them since this is beyond my mental capabilities :anguished: .  I'll be looking forward to the next commit :thumbsup: .